css/style.css

```diff
@@ -1,4 +1,4 @@
-* {
+.section-content {
   font-family: "Gill Sans",Helvetica,sans-serif;
```

Comment:
@seamuslee001 it seems the color property is causing the issue to menu text. In order to retain the font style can you move the ```color``` property to ```.section-content``` as in 

```patch
* {
   ...
  - color: #4B4C4D !important;
}

+ .section-content {
+    color: #4B4C4D !important;
+ }
```